### ISSUE TITLE
Multi-line Julia Code

### ISSUE THREAD

**Issue by BradyAJohnston** [quarto-dev/quarto-vscode#7]
When working in VSCode, multi-line code runs if you run the whole cell, but if the code is split over multiple lines and you use the CMD + Enter / Ctrl + Enter to run a single-line, it only attempts to run the one line and breaks. 

With the chunk below, hitting Ctrl+Enter on the first line:

```julia
randn(
    10
)
```

Results in this error: 
```bash
julia> randn(
ERROR: syntax: incomplete: premature end of input
Stacktrace:
 [1] top-level scope
   @ code:1
```

Multi-line but not whole-chunk execution would be very handy. Thanks for the great plugin so far though!

**Comment by eitsupi** [comment]
I believe this problem also occurs outside of Julia.
I was editing R the other day and needed to run it in code block units.

**Comment by jjallaire** [comment]
Yes in both cases the plugin doesn't have the syntactic awareness to run the next statement (only lines). I believe in the case of R the interpreter will prompt for the next line but in Julia is yields an error.

**Comment by mcanouil** [comment]
I can't verify, but it might be fixed thanks to 0ac9f340a5cf1b9af10aecc097601fb27a98f034 released in v1.54.0.

**Comment by jjallaire** [comment]
No, this doesn't yet work for Julia. I tried to make it work by executing the `language-julia.executeCodeBlockOrSelectionAndMove` command from the Julia extension, however an error occurred (it may be that it didn't like that `qmd` rather that `jl` was where the command was executed from)

**Comment by EKtheSage** [comment]
I asked a similar question in Quarto CLI: https://github.com/quarto-dev/quarto-cli/discussions/3679

Do we have a solution to execute multiline statement like in RStudio?
